### PR TITLE
Add littlefoot to Footnotes block

### PR DESCRIPTION
**Pull request by s3rgiosan**

### CODE REVIEW

File: wp-littlefoot.php [hunk 1]
```diff
@@ -0,0 +1,133 @@
+<?php
+/**
+ * Plugin Name:       littlefoot
+ * Description:       Footnotes without the footprint.
```
Comment (s3rgiosan) [inline wp-littlefoot.php]: @goblindegook This was copied from littelfoot, but we probably need to add more context to what this plugin does to the Footnotes block. Same thing for the "Plugin Name".

File: wp-littlefoot.php [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+ * Plugin URI:        https://github.com/s3rgiosan/wp-littlefoot
+ * Requires at least: 6.1
+ * Requires PHP:      7.4
+ * Version:           0.1.0
+ * Author:            Sérgio Santos
+ * Author URI:        https://s3rgiosan.dev/?utm_source=wp-plugins&utm_medium=wp-littlefoot&utm_campaign=author-uri
+ * License:           MIT
+ * License URI:       https://opensource.org/license/mit/
```
Comment (s3rgiosan) [inline wp-littlefoot.php:12]: @goblindegook I kept the same license from littlefoot

Comment (xipasduarte) [reply wp-littlefoot.php:12]: And can you keep the same license, given it's a WordPress plugin with WordPress code mixed together with rest?

File: wp-littlefoot.php [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+define( 'WP_LITTLEFOOT_PATH', plugin_dir_path( __FILE__ ) );
+define( 'WP_LITTLEFOOT_URL', plugin_dir_url( __FILE__ ) );
+
+if ( file_exists( WP_LITTLEFOOT_PATH . 'vendor/autoload.php' ) ) {
+	require_once WP_LITTLEFOOT_PATH . 'vendor/autoload.php';
+}
+
+/**
+ * Enqueue assets.
```
Comment (xipasduarte) [inline wp-littlefoot.php]:
```suggestion
 * Register assets.
```

**Review by xipasduarte — APPROVED** [review]
Are you planning on extending the block to add littlefoot's options?

**Comment by s3rgiosan** [comment]
> Are you planning on extending the block to add littlefoot's options?

At some extent, yes. In the works.

File: wp-littlefoot.php (hunk 2 above)
Comment (s3rgiosan) [reply wp-littlefoot.php:12]: Gray area? 🤷‍♂️ Really don't know. I'm not planning to release this in wp.org so probably not _a thing_, but https://opensource.stackexchange.com/questions/8520/can-i-publish-a-wordpress-site-as-mit

File: wp-littlefoot.php (hunk 1 above)
Comment (goblindegook) [reply wp-littlefoot.php]: The name of the library refers to its small size, it's not very useful when describing a WordPress plugin. Try this:

> Elegant, interactive popups for footnotes.

**Review by goblindegook — APPROVED** [review]

File: wp-littlefoot.php (hunk 1 above)
Comment (s3rgiosan) [reply wp-littlefoot.php]: How about:
```
 * Plugin Name:       littlefoot
 * Description:       Elegant, interactive popups for the Footnotes block.
```

@goblindegook @xipasduarte

Comment (s3rgiosan) [reply wp-littlefoot.php]: @goblindegook Same as you said, but referring to the Footnotes block explicitly.

**Review by goblindegook — APPROVED** [review]

**Pull request merged**